UrbanAirship.iOS.Accengage.nuspec

```diff
@@ -15,5 +15,7 @@
 
    <files>
      <file src="src/AirshipBindings.iOS.Accengage/bin/release/AirshipBindings.iOS.Accengage.dll" target="lib/Xamarin.iOS10" />
+     <file src="src/AirshipBindings.iOS.Accengage/bin/Release/AirshipBindings.iOS.Accengage.resources/**" target="content/AirshipBindings.iOS.Accengage.resources" />
```

Comment:
This line copies over the "resources" directory to the target. The resources directory contains the xcframework as well as a manifest file (autogenerated) that tells VS how to actually link the xcframework, which contains multiple architecture slices that currently confuse the linker without further instruction. We may be able to remove this at some point in the future.